src/main/java/com/internet/shop/servise/ProductServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.internet.shop.servise;
```

Comment:
```suggestion
package com.internet.shop.servise.impl;
```
let's move implementations to separate sub-package